service/resource/project/resource.go

```diff
@@ -58,13 +65,23 @@ func (r *Resource) GetCurrentState(obj interface{}) (interface{}, error) {
 		return nil, microerror.Mask(err)
 	}
 
-	r.logger.Log("debug", "get current state")
```

Comment:
These logs are kind of redundant because the log resource wraps this resource already and provides exactly this information in a more unified way.